### DISCUSSION TITLE
CSS property `background-color` does not change

### DISCUSSION THREAD

**Discussion by AndreyZakhvatoshin** [gotenberg/gotenberg#444] [Q&A] [answered]
Hello! I use gutenberg to create pdf from blade templates. I want to change the class like this:
`<body class="{{$theme}}" style="--accent-color:{{$color}}">` 
or 
`<body style="background-color: red">`
But background don`t change. Other css properties work.

**Comment by gulien** [comment, marked as answer]
Hello @AndreyZakhvatoshin 

You may try the form field `printBackground` with `true`.

Tip: Gotenberg & Puppeteer uses the same Google Chrome API. Issues found in Gotenberg are often relevant in Puppeteer too, and there are a lot more folks using the later, so more chance to find the culprit :)
